data_host/chrome_extension/src/pages/Popup/index.tsx

```diff
@@ -5,18 +5,27 @@ import { Store } from 'webext-redux';
 
 import { REDUX_PORT_NAME } from '../../redux/port';
 
+import {FractalUI, Background} from '@trustfractal/data-host';
 import Popup from './Popup';
 import './index.css';
 
 const store = new Store({ portName: REDUX_PORT_NAME });
 
 store.ready().then(() => {
-  render(
-    <Provider store={store}>
-      <Popup />
-    </Provider>,
-    window.document.querySelector('#app-container')
-  );
+    let ui = new FractalUI();
+    ui.render(document.getElementById('app-container')).then(() => {
+       console.log('FractalUI render promise called.')
+    });
+
+    let background = new Background();
+    background.setup().then(() => { console.log( 'Background setup promise called.')})
```

Comment:
The chrome extension's "popup" script is not the background context. I believe the background script is specified in the manifest. Clients should call our background initialization in the actual background.